pyproject.toml

```diff
@@ -45,7 +45,7 @@ plugins = "pydantic.mypy"
 [tool.pylint]
 # Ignore too-few-public-methods due to pydantic models
 # Ignore no-self-argument due to pydantic validators
-disable = "wrong-import-order,redefined-outer-name,too-many-instance-attributes,too-few-public-methods,no-self-argument,fixme,parse-error"
+disable = "wrong-import-order,redefined-outer-name,too-many-instance-attributes,too-few-public-methods,no-self-argument,fixme,protected-access"
```

Comment:
i really want to test some protected member functions of the snap class.  So-- lets ignore those errors